book/website/project-design/project-management-methodologies/overview.md

```diff
@@ -0,0 +1,30 @@
+# Project Management Methodologies/Frameworks Overview
+
+Choosing the right project management framework for a research project is crucial, as it can significantly influence your research project's success, efficiency, and potential for collaboration.
+Research projects are often characterised by uncertainty, evolving objectives, and a need for flexibility.
+As such, they require a framework that can accommodate these dynamics.
+As a recent study pointed out, "a fine balance is needed between implementing management approaches to support the efficient and effective delivery of projects while not compromising accountability, creativity and innovation." [santos2022framework]
```

Comment:
Just double-checking if this reference is already in the references or needs to be added?

Reply:
Batool, looks like it might need to be added

Reply:
@Arielle-Bennett I will add it.

Reply:
Thank you so much @Arielle-Bennett and @BatoolMM for doing this in my absence!!!